environment/deployments/science-platform/cloudsql/main.tf

```diff
@@ -222,6 +238,12 @@ resource "google_service_account_iam_member" "vo_cutouts_sa_wi" {
   member             = "serviceAccount:${var.project_id}.svc.id.goog[vo-cutouts/vo-cutouts]"
 }
 
+resource "google_service_account_iam_member" "ssotap_uws_sa_wi" {
+  service_account_id = module.service_accounts.service_accounts_map["ssotap-uws"].name
+  role               = "roles/iam.workloadIdentityUser"
+  member             = "serviceAccount:${var.project_id}.svc.id.goog[ssotap-uws/ssotap-uws]"
+}
```

Comment:
The service account should just be called `ssotap`, since that corresponds to the ssotap Kubernetes application and it will only have a single service account, no matter how many databases it connects to. In the `member` line, also, the part before the slash is the Kubernetes namespace, which is just `ssotap`.

Reply:
Ok makes sense, updated accordingly